### PR TITLE
Add todo-cell sample

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Add todo-cell sample

### CODE REVIEW

File: cells/todo-service/README.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+            PORT: {
+                value: "8080"
+            },
+            DATABASE_HOST: {
+                value: cellery:getHost(mysqlComponent)
+            },
+            DATABASE_PORT: {
+                value: mysqlPort
+            },
+            DATABASE_USERNAME: {
```
Comment (sinthuja) [inline cells/todo-service/README.md]: Do we need USERNAME and PASSWORD as env when it's passed as secret?

File: cells/todo-service/README.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+   cd <SAMPLES_ROOT>/cells/todo-service
+   ```
+
+### 2. Build and run cell
+In this section let's focus on build, run and push a [todo cell](todo-cell.bal). 
+
+The `todo cell` contains two components `todo` and `mysql`. The `todo` component is defined by a container image `docker.io/wso2cellery/samples-todoapp-todos:latest` 
+which is written in Go Lang and it is a simple micro service that add/list/update todo items by connecting to database. The `mysql` component is a MySQL database that is used to 
+persists the todo items received by the `todo` component.
+
```
Comment (sinthuja) [inline cells/todo-service/README.md]: Shall we add a description about the intension of the sample? Ie, volumes such as persistent storage, config maps, and secrets. And then explain a bit about each config segment. Then link to the spec for more information about those each options.

File: cells/todo-service/README.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    
+    What's next?
+    --------------------------------------------------------
+    Execute the following command to run the image:
+      $ cellery run <CELLERY_HUB_ORG>/todo-cell:latest
+    --------------------------------------------------------
+    ```
+2. Once the todo-cell create a volume according to your setup as decribed below. 
+    
+    #### Docker for mac
```
Comment (sinthuja) [inline cells/todo-service/README.md]: I hope you will add relevant commands to other environment options as well.

File: cells/todo-service/README.md (hunk 1 above)
Comment (anuruddhal) [reply cells/todo-service/README.md]: Fixed.

File: cells/todo-service/README.md (hunk 3 above)
Comment (anuruddhal) [reply cells/todo-service/README.md]: Added the other steps.

File: cells/todo-service/README.md (hunk 2 above)
Comment (anuruddhal) [reply cells/todo-service/README.md]: Added to the README.

File: Makefile [hunk 4]
```diff
@@ -26,8 +26,8 @@ CELLS_DIR := cells
 COMPOSITES_DIR := composites
 TESTS_DIR := tests
 SAMPLES_SRC := pet-store hello-world hello-world-api
-SAMPLES_CELLS := pet-store hello-world hello-world-api hipster-shop
-SAMPLES_COMPOSITES := pet-store todo-service hello-world
+SAMPLES_CELLS := pet-store hello-world hello-world-api hipster-shop todo-service
+SAMPLES_COMPOSITES := pet-store todo-service
```
Comment (sinthuja) [inline Makefile]: Can you add back the hello-world composite?

**Pull request merged**